src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleViewLayoutManager.cs

```diff
@@ -708,6 +708,31 @@ protected virtual FlexibleViewViewHolder FindLastVisibleItemView()
             return null;
         }
 
+        /// <summary>
+        /// Dispose FlexibleView and all children on it.
+        /// </summary>
+        /// <param name="type">Dispose type.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                mScrollAni?.Dispose();
+
+                if (mFlexibleView != null)
```

Comment:
ditto

Reply:
Thank you! I will also remove the codes related to mFlexibleView here.